### ISSUE TITLE
i got an error 

### ISSUE THREAD

**Issue by MrAryavart** [emmett1/lfs-scripts#9]
[https://drive.google.com/file/d/1tUqR67hdNJ5PL9lyjqJswwqIhC_Gx4Lt/view?usp=sharing](url)
i have shared the image with google drive link.

**Comment by emmett1** [comment]
The link you gave shows nothing.

**Comment by emmett1** [comment]
Script is updated. If you can confirm its working fine now on Mint i will add it to tested host section.

**Comment by MrAryavart** [comment]
I will check and confirm

**Comment by MrAryavart** [comment]
when i start the toolchain script i get this:
mkdir: cannot create directory '/mnt/lfs': Permission denied
mkdir: cannot create directory '/mnt/lfs': Permission denied

**Comment by MrAryavart** [comment]
the error which i was telling 

collect2: error: ld returned 1 exit status
make: *** [Makefile:46: pkgadd] Error 1
!!! build pkgutils-5.40.7 failed !!!

**Comment by M0E-lnx** [comment]
I got this same error.  It looks like the problem is related to libarchive

The actual error starts with ``pkgutil.cc:(.text+0x1875): undefined reference to archive_read_new`` ans is followed with lots similar messages about all kinds libarchive functions.

No clue how to fix this, but I thought I'd mention what I came across.

**Comment by MrAryavart** [comment]
Let's wait for @emmett1 to reply

**Comment by M0E-lnx** [comment]
BTW, I think this is a duplicate or #7 or at least related to it.

**Comment by emmett1** [comment]
> when i start the toolchain script i get this:
> mkdir: cannot create directory '/mnt/lfs': Permission denied
> mkdir: cannot create directory '/mnt/lfs': Permission denied

Yes this happen to me too but the build still success for me. Sure i will take a look at it later :D

**Comment by emmett1** [comment]
> the error which i was telling
> 
> collect2: error: ld returned 1 exit status
> make: *** [Makefile:46: pkgadd] Error 1
> !!! build pkgutils-5.40.7 failed !!!

What distro as host did you use? And is it live environment or not?

**Comment by emmett1** [comment]
> I got this same error. It looks like the problem is related to libarchive
> 
> The actual error starts with `pkgutil.cc:(.text+0x1875): undefined reference to archive_read_new` ans is followed with lots similar messages about all kinds libarchive functions.
> 
> No clue how to fix this, but I thought I'd mention what I came across.

What distro as host did you use? And is it live environment or not?

**Comment by emmett1** [comment]
> BTW, I think this is a duplicate or #7 or at least related to it.

Those issue already fix in latest update. Make sure you on latest script.

**Comment by MrAryavart** [comment]
@emmett1 Distro is Linux Mint and it was not live

**Comment by MrAryavart** [comment]
i am really sorry i didn't see other issues its same as #7

**Comment by emmett1** [comment]
Debian based distro often give problem when build lfs. Try other distro then tell me the result. I will try on debian distro when i got free time and try to make it work.

**Comment by MrAryavart** [comment]
i am trying on manjaro and i didn't face the pkg build error...

**Comment by emmett1** [comment]
Ok cool, i will add to tested host list.

**Comment by jmarroyave** [comment]
I had the same error on Ubuntu, the problem was because libarchive was missing

sudo apt-get install libarchive-dev

**Comment by voncloft** [comment]
Receiving the smae error pkgutil.cc:(.text+0x295f): undefined reference to `archive_read_new'

Using kubuntu

Tried libarchive-dev installing
